pkg/controllers/clusterresourceplacementeviction/controller_test.go

```diff
@@ -503,16 +530,7 @@ func TestExecuteEviction(t *testing.T) {
 			name: "PickAll CRP, Misconfigured PDB MaxUnavailable specified - eviction not executed",
 			validationResult: &evictionValidationResult{
 				crb: availableBinding,
-				crp: &placementv1beta1.ClusterResourcePlacement{
-					ObjectMeta: metav1.ObjectMeta{
-						Name: testCRPName,
-					},
-					Spec: placementv1beta1.ClusterResourcePlacementSpec{
-						Policy: &placementv1beta1.PlacementPolicy{
-							PlacementType: placementv1beta1.PickAllPlacementType,
-						},
-					},
-				},
+				crp: ptr.To(buildTestPickAllCRP(testCRPName)),
```

Comment:
I wonder why sometimes we use this while sometimes we just use testCRP

Reply:
testCRP is only used for validateEviction test where we use defaulter to set default values on the CRP. For all the other tests we can build CRP which doesn't need to have all default values